tests/cassettes/test_integration/test_pypi_error.yaml

```diff
@@ -0,0 +1,308 @@
+        kb6Vnl+Rk5PgUk5HiS+LTW4xwXSrTxsxQldOiNyL698HEIFJgSMmDySnHtMlABEiwNMMyWdUqvSh
+        oWxN+eMQBzfClgfatmjKr7qnyhag5YDQgd8rtJdzhFQSBwFovFLCzHg7fjFlJtaQM9NlEUnkVaeJ
+        F7vAgGqKQRX9tio+UtmjrHgJz5QR8pIAKWNHkkB2lEyP2jgw486nFAnQ9wUqRjMVcAAnsPAKTad6
+        BsGCjfYTvxugeaJoKmK4WzH8n9JEdnqUiT2VgphIDsCqDf4TAwB5hov1ZilEmcnSJx2UqUy1rADH
+        RChlwH3jSEWRHQD7HpU+4wMMGMKs8gkGM3hWH2AwSSx8gsFsbXyAQfd4/BMMuY60E8+faDnVjk3r
+        N1P/44zGa6r/bmzl3v+i77/s7TaK+p8Xef9L27kbS6UopctnzfzLG9d8PB64/ii4sWJ7PJDvgTXZ
+        o99/wUZDdKVN1rCKFzl+S/V/hlysRf/xa1+L9X+F/r/QdRSE88gdT2K24Wwy/O4WufwH71nILyy5
+        8ZxtgGtvbVrWJZZfySocVwbld3MI2W2fsMko4hQE4eE0FXcFADwArgMEwWKOO4QjVCgPiC7Ecg1L
+        VurrExZC7EIEDuLxYf5FBhVqbcjyFtWjtEmDDDke7vsWPkuPa7AACQIzs8TKLArXjz2Iz9UI2J3Y
+        gogKi+wrNM8K1kQCgIOfnJal6tkqRtKygmlgZUsJbdUR+HHMqgWhC/GaOgfVs1MlCwHyBsZXLCIk
+        N5uoUsJ0JYiDk8iHISX8GgaWCGhE/WZDPt5w8ISTUO2hfG+Bsgy0FrnrfoDvqqjCKIwisl1Vj8TE
+        1pFjFuTDLUsuB494YXhQDz/G92wQxMkStvwyser/fZv1Lk76H1rdNuv02GX34ufOcfuYlVo9+L1U
+        YR86/fcXV30GLbqt8/5HdnHCWucf2U+d8+MKa//XZbfd67GLrtU5uzzttOFe5/zo9Oq4c/6OvYV+
+        5xd9cFtnnT4Q7V8wHFCR6rR7SOys3T16D7+23nZOO/2PFeuk0z9HmicXXdZil61uv3N0ddrqssur
+        7uVFrw3DHwPZ8875SRdGaZ+1z/s1GBXusfbP8AvrvW+dnuJQVusKZt/F+bGji8uP3c679332/uL0
+        uA0337ZhZq23p205FCzq6LTVOauw49ZZ612bel0Ala6FzeTs2If3bbyF47Xgz1G/c3GOyzi6OO93
+        4dcKrLLbT7t+6PTaFdbqdnrIkJPuxVnFQnZCjwsiAv3O25IKsprldgSa4O9XvXZKkB23W6dAq4ed
+        cYm6ca1w6b9b/09R8SpjrP79z6397YPi+5+/5P1Ps39riP8aW4v4b2d3b6vAfy9xUYIbN7MwhYX9
+        f2b+J3JW+wb0V9j/vcZeYf9/wftP3Qa5bjU+HlcxQ1T/9v0/2N4r9v9Xu/+LHwhf+fvfO40d3P/C
+        /3//q/j+d/H97+L738X3v4vvfxff/y6+/118/7v4/nfx/e/i+9/F97+L738X+b+vjf97F1fdo3Yv
+        LRB5Kv+/v79Q/7WztVXk/1/mMou8jA87W+lHiayFLwOn1X+WxtpW/kDIQvmQklEfDFwICgcDbGXc
+        XqzWN5+pl8pUh2flmaxVJPLpxukrJBDMuhiLP6uXWWT9vB761Y/ntY6DcOChE3t+3dX/l/4vsuLJ
+        87+9hfzvzvbebvHf/3uR63qh8P/GUtFrkxml/kUFZ+H/n6//pmF71vn//mL+f2u/+P7/y1zZS5mW
+        Jd9YzL8hCHcRVcNt9VJgYQkK/V8Rqjyp/42DRf/faBT//Z+Xqf+RB0yFIvxOr1q9Vv/raeCPTyHU
+        +U5jLKrGws8GBvtmLUhja2t7a/sVO31J/V/3In/H9n9ZAF1o2u93/5/2/1sP4v9GUf/zIlfh+Qv9
+        /wr9V8na57vG1eo/tw/294v6z1/+/hv5/SdY88X3//cOFt7/2NtqFOc/L3IN+Yg+D7KxKb+UUiqV
+        jA/rpDUfxler1Qd88D/OIT/vAe02Skf4kiiVOdGHj8329IlELEQpbRbe5rel/4sHeV+n/w/f/9pv
+        7BT535fJ/xTvfxX6/9X6nx7WP4n/vnz+u7Og/7v728X574v5f3s4HAQ+3/CT6R2PFBCIeJxEPpP3
+        2H+yrcJ3F1dxFVdxFVdxFVdxFVdxFVdx/Zqv/wN2gOlbAKAAAA0KLS0xYzcwMTIzOGVkZmM0ZjRh
+        YjZkMjU0NDU1ZjkxZGU1Mi0tDQo=
+      - 0
+      - null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - Basic Zm9vOmJhcg==
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '12731'
+      Content-Type:
+      - multipart/form-data; boundary=1c701238edfc4f4ab6d254455f91de52
+      User-Agent:
+      - twine/3.2.1.dev15+g87846e5.d20201003 pkginfo/1.5.0.1 requests/2.24.0 setuptools/40.6.2
```

Comment:
When we change the version, will this break?

Reply:
I don't think so, based on the [default request matching](https://vcrpy.readthedocs.io/en/latest/configuration.html#request-matching):

>  The default behavior is `['method', 'scheme', 'host', 'port', 'path', 'query']` which means that requests with both the same URL and method (ie POST or GET) are considered identical.